eth-types/src/geth_types.rs

```diff
@@ -402,3 +412,18 @@ impl GethData {
     }
 }
 */
+
+/// Return access list address size and all storage key size.
+pub fn access_list_address_and_storage_key_sizes(access_list: &Option<AccessList>) -> (u64, u64) {
```

Comment:
```suggestion
/// Returns the number of addresses and the cumulative number of storage keys in the entire access list.
pub fn access_list_size(access_list: &Option<AccessList>) -> (u64, u64) {
```

Reply:
Rename the function and comment with other used places in commit [1988777](https://github.com/scroll-tech/zkevm-circuits/pull/1045/commits/198877743211fbaef748891523986f5ace1c3293), thanks.